src/test/java/com/rocketden/main/util/Utility.java

```diff
@@ -9,4 +9,8 @@ public static String convertObjectToJsonString(Object o) {
         return gson.toJson(o);
     }
 
+    public static <T> T toObject(String json, Class<T> c) {
```

Comment:
Used the Gson codebase as a reference for this; essentially, the generics allow you to specify a class as the second parameter, and this method will then convert the JSON string into that class and return it.

Reply:
That's awesome, nice job on finding this 👍